lib/common/media/media.types.ts

```diff
@@ -24,48 +24,73 @@ import { MediaEntity } from "./entity/media.entity";
  * Interface representing a media object.
  */
 export interface Media {
+  /** This field contains the identification number of the file  */
   id: number;
+  /** This field contains a unique character identifier */
   code: string;
+  /** This field contains the name of the file in all languages involved */
   name: LocalizedString[];
+  /** This field contains a list of unique settings and functions that the file contains */
   type: MediaType;
+  /** This field contains the media file and all copies and thumbnails */
   files: MediaFile[];
+  /** This field contains a link to the meta data container */
   metadata: FileMetadata;
+  /** This field contains the date the file was created */
   tsCreated: Date;
 }
 
 /**
  * Interface representing a media type.
  */
 export interface MediaType {
+  /** A unique code representing the media file type */
   code: string;
+  /** This field contains a unique character identifier */
   name: string;
+  /** Indicates whether the media type supports the VP6 video codec */
```

Comment:
add that this field defines the possibility of creating a duplicate file in WEBP format